api/app/src/models/medical/patient.ts

```diff
@@ -1,16 +1,41 @@
 import { DataTypes, Sequelize } from "sequelize";
 import { getOrganizationOrFail } from "../../command/medical/organization/get-organization";
 import { Config } from "../../shared/config";
-import { OIDNode, OID_ID_START } from "../../shared/oid";
+import { OIDNode, OID_ID_START, USState } from "../../shared/oid";
 import { BaseModel, defaultModelOptions, ModelSetup } from "../_default";
+import { ExternalMedicalPartners } from "./../../external";
+import { Address } from "./address";
+import { Contact } from "./contact";
+
+export abstract class PatientDataExternal {}
+
+export type DriversLicense = {
+  value: string;
+  state: USState;
+};
+
+export type PatientData = {
+  firstName: string;
+  lastName: string;
+  dob: string;
+  gender: string; // TODO #369 do we need to support CW SDK's genderSchema?
```

Comment:
@Goncharo @Orta21 do you guys think we need to implement a more complex gender schema throughout our API and Dash, like CW SDK's `genderSchema`, or can we just use a simplified version like this (API schema: `patientCreateSchema`)?